pom.xml

```diff
@@ -218,12 +268,23 @@
 
     <properties>
         <!-- Carbon Kernel -->
-        <carbon.kernel.version>4.10.9</carbon.kernel.version>
+        <carbon.kernel.version>4.10.14</carbon.kernel.version>
         <maven.checkstyle.plugin.version>3.1.0</maven.checkstyle.plugin.version>
         <maven.spotbugs.plugin.version>4.2.3</maven.spotbugs.plugin.version>
+        <spotbugs.annotations.version>4.7.3</spotbugs.annotations.version>
         <com.h3xstream.findsecbugs.version>1.12.0</com.h3xstream.findsecbugs.version>
         <commons.logging.version>1.2</commons.logging.version>
         <axiom.wso2.version>1.2.11-wso2v16</axiom.wso2.version>
+        <commons-lang.version>3.4</commons-lang.version>
+        <json.wso2.version>3.0.0.wso2v4</json.wso2.version>
+        <com.googlecode.json.version>1.1.1</com.googlecode.json.version>
+        <commons-collections.version>4.4</commons-collections.version>
+        <org.testng.version>7.7.0</org.testng.version>
+        <jacoco.version>0.8.6</jacoco.version>
+        <commons-dbcp.version>1.4</commons-dbcp.version>
+        <orbit.version.h2.engine>1.2.140.wso2v3</orbit.version.h2.engine>
+        <mockito.version>1.10.19</mockito.version>
+        <powermock.version>1.7.1</powermock.version>
```

Comment:
cant we use only mockito like we discussed offline?

Reply:
Powermock allows to mock static methods once per class. But in mockito we need to mock in each test. Hence used powermock to mock static methods.